cartographer/mapping/id.h

```diff
@@ -225,6 +235,11 @@ class MapById {
     return IdType{trajectory_id, index};
   }
 
+  // Returs an iterator to the element at id or throws an exception.
```

Comment:
typo: `Returs` -> `Returns`
`id` -> `'id'`
We use a `CHECK`, so: `or throws an exception` -> `which must exist`.

Reply:
Done